plugins/node/opentelemetry-instrumentation-mysql/src/instrumentation.ts

```diff
@@ -323,6 +332,11 @@ export class MySQLInstrumentation extends InstrumentationBase<
     };
   }
 
+  private _defaultDbStatementSerializer(dbStatement: string) {
+    //replace all quoted words with a question mark (' and ". not `)
```

Comment:
If I understand correctly, then this sql query:
```SQL
SELECT * FROM "customers"
```

Will turn into 
```SQL
SELECT * FROM ?
```
?

Reply:
right